### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by michaltakac** [GravityProject/gravity#32]
Should we just quickly prototype and start testing later?
Should we prototype without testing?
Any thoughts about testing appreciated (how, what to use, Travis CI and so on).

**Comment by pmuens** [comment]
Testing would be great, but I don't have any experiences with it so far (I have tinkered around with mocha and jasmine).

Do you have any experiences with Meteor and testing? Travis CI would be great as well.

**Comment by michaltakac** [comment]
Not much but have already setup Travis CI for one of my side-projects. This project seems like great opportunity to dive into tsting a bit further!

**Comment by pmuens** [comment]
Yes, maybe we can start with some tests but also prototype without testing so that we can implement new features faster.
